go/client/chat_cli_rendering.go

```diff
@@ -515,14 +515,18 @@ func formatSendPaymentMessage(g *libkb.GlobalContext, body chat1.MessageSendPaym
 		return "[error getting payment details]"
 	}
 
-	ls := strings.ToLower(details.Status)
-	if ls != "completed" && ls != "pending" {
-		return fmt.Sprintf("error sending payment: %s", details.StatusDetail)
-	}
-
-	verb := "sent"
-	if ls == "pending" {
+	var verb string
+	statusStr := strings.ToLower(details.Status)
+	switch statusStr {
+	case "completed", "canceled", "claimable":
+		verb = "sent"
+		if details.Yanked {
+			verb = "canceled sending"
```

Comment:
the verb for yanked is canceled but the verb for canceled is sent?

Reply:
i don't think "canceled" gets plumbed there, hence the need for another field in struct. It's also "completed" when it's yanked, at least from hwat I observed

Reply:
Ok. Everything about the CLI transform and CLI avdl types is for the CLI and not shared